charts/backend/templates/deployment.yaml

```diff
@@ -101,18 +101,38 @@ spec:
 {{- if ne (len .Values.extraVolumeMounts) 0 }}
 {{ toYaml .Values.extraVolumeMounts | indent 12 }}
 {{- end }}
+
+{{ if .Values.probes.backend }}
```

Comment:
I think we can skip this if statement. The values.yaml sets .Values.probes.backend by default, so this will always evaluate to true.